src/schema/resolvers.ts

```diff
@@ -7,6 +7,10 @@ import jwt from 'jsonwebtoken';
 const resolvers = {
   Query: {
     hello: () => 'Hello, World!',
+    user: async (_, { id }, context) => {
+      authorizeAccess(context);
+      return await getUserById(id);
```

Comment:
```suggestion
      return getUserById(id);
```

O `getUserById` já retorna uma Promise, então não precisa do await